sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -0,0 +1,542 @@
+    options!.requestPolicyFactories = requestPolicyFactories;
+    options = {
+      requestPolicyFactories: requestPolicyFactories
+    };
+
+    super(credentials, options);
+    this.queueResourceSerializer = new QueueResourceSerializer();
+    this.topicResourceSerializer = new TopicResourceSerializer();
+    this.subscriptionResourceSerializer = new SubscriptionResourceSerializer();
+    this.ruleResourceSerializer = new RuleResourceSerializer();
+    this.endpoint = (connectionString.match("Endpoint=sb://(.*).servicebus.windows.net") || "")[1];
+  }
+
+  /**
+   * Creates a queue.
+   *
+   * @param {string}             queuePath                                             A string object that represents the name of the queue to create.
+   * @param {object}             [options]                                             The request options.
+   * @param {int}                [options.MaxSizeInMegaBytes]                          Specifies the maximum queue size in megabytes. Any attempt to enqueue a message that will cause the queue to exceed this value will fail.
+   * @param {PTnHnMnS}           [options.DefaultMessageTimeToLive]                    Depending on whether DeadLettering is enabled, a message is automatically moved to the DeadLetterQueue or deleted if it has been stored in the queue for longer than the specified time. This value is overwritten by a TTL specified on the message if and only if the message TTL is smaller than the TTL set on the queue. This value is immutable after the Queue has been created.
+   * @param {PTnHnMnS}           [options.LockDuration]                                Determines the amount of time in seconds in which a message should be locked for processing by a receiver. After this period, the message is unlocked and available for consumption by the next receiver. Settable only at queue creation time.
+   * @param {bool}               [options.RequiresSession]                             Settable only at queue creation time. If set to true, the queue will be session-aware and only SessionReceiver will be supported. Session-aware queues are not supported through REST.
+   * @param {bool}               [options.RequiresDuplicateDetection]                  Settable only at queue creation time.
+   * @param {bool}               [options.DeadLetteringOnMessageExpiration]            This field controls how the Service Bus handles a message whose TTL has expired. If it is enabled and a message expires, the Service Bus moves the message from the queue into the queue’s dead-letter sub-queue. If disabled, message will be permanently deleted from the queue. Settable only at queue creation time.
+   * @param {bool}               [options.DuplicateDetectionHistoryTimeWindow]         Specifies the time span during which the Service Bus detects message duplication.
+   * @param {bool}               [options.EnablePartitioning]                          Specifies whether the queue should be partitioned.
```

Comment:
@ramya-rao-a can you clarify if this hurts or helps docs rendering as such?
Looks like a lot of effort already went into the alignment, we should probably keep it unless there are any issues.

Reply:
If this is what exists today for `azure-sb`, then you can check out the docs for it at https://docs.microsoft.com/en-us/javascript/api/azure-sb/?view=azure-node-latest to see how it gets rendered

Reply:
Even if we go with the argument that the spacing was done to have vertical alignment among the param descriptions, the spacing in rest of the methods in the file other than the create methods is way too much. So that definitely must be re-visited.